presto-spark-base/src/test/java/com/facebook/presto/spark/TestPrestoSparkQueryRunner.java

```diff
@@ -630,6 +631,36 @@ public void testSmileSerialization()
         }
     }
 
+    @Test
+    public void testMissingPipelineStatistics()
```

Comment:
This doesn't check the generated statistics or the statistics at each potential level of compaction. Arguably maybe it errors out on query completed with one of the levels of compaction and we don't know?

Reply:
It tests with the maximum level of compaction. Though in theory it is possible that one level before max compaction could potentially be causing problems, but I assume that it is rather low probability.